bokehjs/src/lib/core/util/menus.ts

```diff
@@ -211,24 +211,21 @@ export class ContextMenu { //extends DOMComponentView {
     if (this.items.length == 0) {
       return
     }
-
-    if (!this._open) {
-      this.render()
-      if (this.shadow_el.children.length == 0) {
-        return
-      }
-      (this.target.shadowRoot ?? this.target).appendChild(this.el)
-      this._position(at ?? {left: 0, top: 0})
-      this._listen()
-      this._open = true
+    this.render()
+    if (this.shadow_el.children.length == 0) {
+      return
     }
+    (this.target.shadowRoot ?? this.target).appendChild(this.el)
+    this._position(at ?? {left: 0, top: 0})
+    this._listen()
+    this._open = true
```

Comment:
Is showing unconditionally just a simplification or is there some bug or need?

Reply:
This change allows to reposition an existing menu if already open.